sdk/cosmos/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/converter/MappingCosmosConverterUnitTest.java

```diff
@@ -64,9 +67,7 @@ public void convertDocumentToAddressCorrectly() {
         objectNode.put(TestConstants.PROPERTY_STREET, TestConstants.STREET);
         objectNode.put(TestConstants.PROPERTY_ID, TestConstants.POSTAL_CODE);
 
```

Comment:
its was unclear to me why writeJsonNode was being called with a ObjectNode instance instead of just passing the objectNode straight to the read method. 

This started failing because writeJsonNode is now using CosmosEntityInformation to check if the entity passed is versioned. When an ObjectNode object is passed to writeJsonNode, CosmosEntityInformation fails to initailize properly because the type of object node is not an entity with proper annotations or fields that match expectations. Specifically, CosmosEntityInformation expects the class to have a field named id or a field annotated with @id. ObjectNode obviously has neither of these.

Unless i am missing something about the purpose of calling writeJsonNode here, i dont think it will ever be used in this manner and the input object should be a proper entity that can be use with CosmosEntityInformation.

Reply:
@Blackbaud-EricSlater - Yups makes sense not to call `writeJsonNode` here.